packages/d3fc-series/README.md

```diff
@@ -429,6 +429,20 @@ The SVG and canvas implementations do not support this property.
 
 If *width* is specified, sets the line width and returns this series. If *width* is not specified, returns the current line width.
 
+<a name="seriesLine_equals" href="#seriesLine_equals">#</a> *seriesLine*.**equals**(*equals*)
+
+The SVG and canvas implementations do not support this property.
+
+If *equals* is specified, sets the equality function used to compare *previousData* with *data*. The result of this check is used to control whether the data is reprojected and rescaled. If *equals* is not specified, returns the current equality function which defaults to always returning false indicating the data has changed.
+
+<a name="seriesLine_scaleMapper" href="#seriesLine_scaleMapper">#</a> *seriesLine*.**scaleMapper**(*scaleMapper*)
+
+The SVG and canvas implementations do not support this property.
+
+If *scaleMapper* is specified, sets the function used to map first the xScale and then the yScale onto matched pairs of JavaScript and WebGL implementations. If *equals* is not specified, returns the current scale mapper which defaults to [webglScaleMapper](https://github.com/d3fc/d3fc/packages/d3fc-webgl#webglScaleMapper).
+
+The returned JavaScript scale is equlity checked using a reference comparison to determine whether the data needs to be rescaled. If the reference is the same as the previous render, the values associated with the scale are not rescaled.
```

Comment:
`equlity` -> `equality`

I assume this will need to change everywhere this sentence is copied

Reply:
🤦‍♂